pkg/crc/preflight/preflight_checks_common.go

```diff
@@ -88,14 +89,23 @@ func fixBundleExtracted() error {
 		return fmt.Errorf("Cannot create directory %s: %v", bundleDir, err)
 	}
 
-	if !crcos.FileExists(constants.DefaultBundlePath) {
-		return fmt.Errorf("%s not exists", constants.GetDefaultBundle())
-	}
-
 	_, err := bundle.Get(constants.GetDefaultBundle())
 	if err != nil {
+		if _, err := os.Stat(constants.DefaultBundlePath); os.IsNotExist(err) {
+			data, err := embed.ReadEmbededFile("bundle-info.json")
+			if err != nil {
+				return err
+			}
+			url, sha256sum, err := bundle.GetBundleURLAndSha256sum(data)
+			if err != nil {
+				return err
+			}
+			if _, err := download.Download(url, constants.DefaultBundlePath, 0664, []byte(sha256sum)); err != nil {
```

Comment:
`GetBundleURLAndsha256sum` could return a []byte directly to avoid the cast. A bit worried there's going to be some mismatch between `"0123ABCD"` read as a string from the json file, and `[]uint8{0x01, 0x23, 0xab, 0xcd}` expected by the download code?